File: mycollab-web/src/main/java/com/esofthead/mycollab/module/crm/events/CrmEvent.java
package com.esofthead.mycollab.module.crm.events;

import com.esofthead.mycollab.eventmanager.ApplicationEvent;

@SuppressWarnings("serial")
public class CrmEvent {
	
	public static class GotoHome extends ApplicationEvent {
		public GotoHome(Object source, Object data) {
			super(source, data);
		}
	}
}

File: mycollab-web/src/main/scala/com/esofthead/mycollab/module/crm/events/CrmEvent.scala
package com.esofthead.mycollab.module.crm.events

import com.esofthead.mycollab.eventmanager.ApplicationEvent

/**
 * @author MyCollab Ltd.
 * @since 5.0.3
 */
object CrmEvent {

  class GotoHome(source: AnyRef, data: AnyRef) extends ApplicationEvent(source, data){}
}
